ITC205_BaseLine_Code/src/Loan.java

```diff
@@ -3,36 +3,36 @@
 import java.util.Date;
 
 @SuppressWarnings("serial")
-public class loan implements Serializable {
+public class Loan implements Serializable {
 	
-	public static enum LOAN_STATE { CURRENT, OVER_DUE, DISCHARGED };
+	public static enum LoanState { CURRENT, OVER_DUE, DISCHARGED };
 	
 	private int ID;

```

Comment:
D1: Lines 10-13, fix variables to be more meaningful 
ID = loanId
B = book
M = member
D = dueDate

Reply:
Example. loanBook, loanMember, loanDueDate